goTunnels/access_scopes.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-package tunnels
+package goTunnels
```

Comment:
When I tried publishing the module it failed since the package crawler only looks at the root of the github repo for the go.mod file. So that had to be moved. The issue then is that the go package naming follows the convention that the package name is the name of the folder it is in so we have to rename the folder to something other than just "go". With these changes the user would just refer to the package as github.com/microsoft/dev-tunnels/gotunnels. Ideally, the name would not contain "go" but that would make the file structure a bit confusing. One alternative could also be naming the package github.com/microsoft/dev-tunnels/go/tunnels and setting up the file structure that way

Reply:
I just made another update with a better structure. This keeps the package name as tunnels and it will be referenced as github.com/microsoft/dev-tunnels/go/tunnels

Reply:
This is what it would look like when published https://pkg.go.dev/github.com/jfullerton44/jake-tunnels@v0.1.0/go/tunnels

Reply:
Can go.mod be in the repo root but still point to `module github.com/microsoft/dev-tunnels/go/tunnels`?

Reply:
Unfortunately, the go mod must be named after its exact location

Reply:
The ssh package follows a similar pattern, except they use /x instead of /go https://pkg.go.dev/golang.org/x/crypto/ssh